pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -604,6 +604,12 @@ func (m *kubeGenericRuntimeManager) SyncPod(pod *v1.Pod, _ v1.PodStatus, podStat
 		podIP = podStatus.IP
 	}
 
+	rawHostIP, err := m.runtimeHelper.GetHostIP()
+	hostIP := rawHostIP.String()
```

Comment:
I think we're using `CachedNodeInfo` here so `nodeInfo` may already be cached in memory. I had thought we were doing a GET to the API every time.